roc/receiver.go

```diff
@@ -34,6 +36,44 @@ func OpenReceiver(rocContext *Context, receiverConfig *ReceiverConfig) (*Receive
 	return (*Receiver)(receiver), nil
 }
 
+func (r *Receiver) Bind(portType PortType, proto Protocol, a *Address) error {
+	errCode := C.roc_receiver_bind(
+		(*C.roc_receiver)(r),
+		(C.roc_port_type)(portType),
+		(C.roc_protocol)(proto),
+		a.raw)
+	if errCode == 0 {
+		return nil
+	}
+	if errCode < 0 {
+		return ErrInvalidArgs // roc_receiver_bind returns negative in 3 cases: invalid args, cannot bind address and not enough resources, maybe we should consider renaming this error to a more generic name?
+	}
+	panic(fmt.Sprintf(
+		"unexpected return code %d from roc_receiver_bind()", errCode))
+}
+
+func (r *Receiver) ReadFloats(floats []float32) error {
```

Comment:
We should check if `floats` is an empty or nil slice. We can panic or silently return in this case. I think we should mimic io.Reader here, i.e. some of its common implementations.

Reply:
not sure which behaviour io.Reader has, but I return ErrInvalidArgs if floats is nil